modules/internal/ChapelTuple.chpl

```diff
@@ -76,18 +92,29 @@ module ChapelTuple {
   //
   // isTuple, isTupleType and isHomogeneousTuple param functions
   //
+  pragma "no doc" // we are not advertising isXxxValue() functions at present
   proc isTupleValue(x: _tuple) param
     return true;
   
+  pragma "no doc"
   proc isTupleValue(x) param
     return false;
-
+  
+  pragma "no doc"
   proc isHomogeneousTupleValue(x: _tuple) param
     return __primitive("is star tuple type", x);
   
+  /*
+    Returns `true` if its argument is a tuple type.
+    The argument must be a type.
+  */
   proc isTupleType(type t) param
```

Comment:
I think we've been no-doc'ing the is*Type functions too, and using isTuple, isHomogeneousTuple as the public facing functions.